package.json

```diff
@@ -24,6 +24,8 @@
     "link-aa-win": "node ./scripts/link-bin.js packages/amplify-app/bin/amplify-app amplify-app-dev",
     "link-dev": "cd packages/amplify-cli && ln -s \"$(pwd)/bin/amplify\" \"$(yarn global bin)/amplify-dev\" && cd -",
     "link-win": "node ./scripts/link-bin.js packages/amplify-cli/bin/amplify amplify-dev",
+    "lint": "yarn eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint-check": "yarn eslint . --ext .js,.jsx,.ts,.tsx --max-warnings=1085",
```

Comment:
Do we need `lint` ?

Reply:
for local development we might want to add
`yarn check-static-analysis` or something like this which will run lint, lint package json and prettier as one command.